plugins/org.eclipse.n4js/src/org/eclipse/n4js/external/NpmManager.java

```diff
@@ -461,26 +486,20 @@ private IStatus uninstallDependenciesInternal(Collection<String> packageNames, f
 						affectedEclipseProjects.addAll(collection);
 					}
 				}
-			});
+			}
 
-			logger.logInfo("Dependency changes have been successfully calculated.");
 			monitor.worked(2);
-
-			logger.logInfo(LINE_SINGLE);
-			logger.logInfo("Scheduling build of affected projects... [4 of 4]");
 			monitor.setTaskName("Scheduling build of projects... [4 of 4]");
 
-			scheduler.scheduleBuildIfNecessary(affectedEclipseProjects);
+			// scheduler.scheduleBuildIfNecessary(affectedEclipseProjects);
```

Comment:
is it some `TODO` to followup on?

Reply:
This is done in `externalLibraryWorkspace.registerProjects(...)`. However, yes, this will also be changed again in a follow-up.